.prettierrc

```diff
@@ -1,24 +1,24 @@
 {
-  "overrides": [
-    {
-      "files": [
-        "*.js",
-        "*.ts",
-        "*.tsx"
-      ],
-      "options": {
-        "trailingComma": "es5",
-        "tabWidth": 4,
-        "useTabs": false,
-        "singleQuote": true,
-        "printWidth": 120
-      }
-    },
-    {
-      "files": "*.json",
-      "options": {
-        "tabWidth": 2
-      }
-    }
-  ]
+    "overrides": [
+        {
+            "files": [
+                "*.js",
+                "*.ts",
+                "*.tsx"
+            ],
+            "options": {
+                "trailingComma": "all",
```

Comment:
Switched to `all` for trailing commas. Are you ok with this?

Reply:
yeap, this is fine :+1: